### PR TITLE
Added ecs notify resource

### PR DESCRIPTION
**Pull request by thullSL**

### CODE REVIEW

**Comment by lclarksl** [comment]
Why has console.log been changed to console.error?

**Comment by thullSL** [comment]
Concourse gets it's return value from running the script by reading stdout which is where console.log goes. console.error goes to sterr.

File: concourse/resources/ecsNotify/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node:4.3.2-wheezy
```
Comment (csscottlogic) [inline concourse/resources/ecsNotify/Dockerfile]: This should probably be just node:4.4.0 now?

**Comment by csscottlogic** [comment]
If something's not an error, it should be logged with console.log, shouldn't it? The return value from the container is the return value of the process with PID 1, and shouldn't depend on anything written to STDOUT?

**Comment by thullSL** [comment]
From [here](https://concourse.ci/implementing-resources.html)

> A resource type's check script is invoked to detect new versions of the resource. It is given the configured source and current version on stdin, and must print the array of new versions, in chronological order, to stdout.

**Comment by csscottlogic** [comment]
That's only the check script?

File: concourse/resources/ecsNotify/resource/out [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+exec 3>&1 # make stdout available as fd 3 for ehe result
```
Comment (csscottlogic) [inline concourse/resources/ecsNotify/resource/out]: "ehe" -> "the"

**Comment by csscottlogic** [comment]
:shipit:
That looks much better. The monkeying with file descriptors is localised and commented, and the script itself uses ```console.log```, as you'd expect.

**Pull request merged**